### ISSUE TITLE
Abort current movement

### ISSUE THREAD

**Issue by cartfjord** [synthetos/TinyG#230]
Hi! I am trying to map the stepper motors to a keyboard at my computer. For example when I press the Down key I want to go in the positive x-direction, until the key is released, then I want to abort the current action. Is there any command to abort the current action over serial? For example if I send

G0 X100 Y0 Z0

but before the motors have reached (100, 0, 0) I change my mind and want to go to (80, 0, 0) instead.

In other words:

G0 X100 Y0 Z0
<Abort>
G0 X80 Y0 Z0 

I am looking for the <Abort> command. 

Thanks in advance
- Cartfjord

**Comment by cartfjord** [comment]
I solved it myself. It is clearly stated how to achieve this functionality here:
https://github.com/synthetos/TinyG/wiki/TinyG-Feedhold-and-Resume
